neo/Consensus/ConsensusContext.cs

```diff
@@ -110,6 +112,25 @@ public void Deserialize(BinaryReader reader)
             if (TransactionHashes.Length == 0 && !RequestSentOrReceived)
                 TransactionHashes = null;
             Transactions = transactions.Length == 0 && !RequestSentOrReceived ? null : transactions.ToDictionary(p => p.Hash);
+            SenderFee = new Dictionary<UInt160, BigInteger>();
+            foreach (Transaction tx in Transactions.Values)
```

Comment:
`Transactions` could be `null` here.

Reply:
> `Transactions` could be `null` here.

Have modified accordingly, thanks so much for the insight:)